proto-list-final.yaml

```diff
@@ -6952,357 +6938,357 @@ validated: null
+kind: VMwareEnginePrivateCloud
 protopath: google.cloud.vmwareengine.v1.PrivateCloud
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: Cluster
-kind: VmwareengineCluster
+kind: VMwareEngineCluster
 protopath: google.cloud.vmwareengine.v1.Cluster
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: Node
-kind: VmwareengineNode
+kind: VMwareEngineNode
 protopath: google.cloud.vmwareengine.v1.Node
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: ExternalAddress
-kind: VmwareengineExternalAddress
+kind: VMwareEngineExternalAddress
 protopath: google.cloud.vmwareengine.v1.ExternalAddress
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: Subnet
-kind: VmwareengineSubnet
+kind: VMwareEngineSubnet
 protopath: google.cloud.vmwareengine.v1.Subnet
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: ExternalAccessRule
-kind: VmwareengineExternalAccessRule
+kind: VMwareEngineExternalAccessRule
 protopath: google.cloud.vmwareengine.v1.ExternalAccessRule
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: LoggingServer
-kind: VmwareengineLoggingServer
+kind: VMwareEngineLoggingServer
 protopath: google.cloud.vmwareengine.v1.LoggingServer
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: NodeType
-kind: VmwareengineNodeType
+kind: VMwareEngineNodeType
 protopath: google.cloud.vmwareengine.v1.NodeType
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: HcxActivationKey
-kind: VmwareengineHcxActivationKey
+kind: VMwareEngineHcxActivationKey
 protopath: google.cloud.vmwareengine.v1.HcxActivationKey
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: DnsForwarding
-kind: VmwareengineDnsForwarding
+kind: VMwareEngineDNSForwarding
 protopath: google.cloud.vmwareengine.v1.DnsForwarding
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: NetworkPeering
-kind: VmwareengineNetworkPeering
+kind: VMwareEngineNetworkPeering
 protopath: google.cloud.vmwareengine.v1.NetworkPeering
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: NetworkPolicy
-kind: VmwareengineNetworkPolicy
+kind: VMwareEngineNetworkPolicy
 protopath: google.cloud.vmwareengine.v1.NetworkPolicy
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: ManagementDnsZoneBinding
-kind: VmwareengineManagementDnsZoneBinding
+kind: VMwareEngineManagementDnsZoneBinding
 protopath: google.cloud.vmwareengine.v1.ManagementDnsZoneBinding
 validated: null
 ---
 service: vmwareengine
 package: google.cloud.vmwareengine.v1
 proto: VmwareEngineNetwork
-kind: VmwareengineVmwareEngineNetwork
+kind: VMwareEngineVmwareEngineNetwork
```

Comment:
`VmwareEngineNetwork`?

Reply:
I think it should be VMwareEngine: https://cloud.google.com/vmware-engine?hl=en

Reply:
The proto resource is "VmwareEngineNetwork". Currently the kind is a bit too verbose with duplicated "VmwareEngine".

Reply:
Oooh gotcha!
I'm okay with either: (1) using `[Service][Resource]` consistently is good for automation and minimizes manual work; (2) removing the duplicate `VMwareEngine` is good for readability. If the duplication can be easily identified and removed, we can do (2); otherwise I'd prefer (1).

And regardless, it should probably be (capital)V(capital)Mware.